### PR TITLE
Added shouldNotBeInRange for ClosedRange #119

### PR DESCRIPTION
**Pull request by Kshitij09**
Signed-off-by: Kshitij09 <kshitijpatil98@gmail.com>

# Description
Added `shouldNotBeInRange` as per the discussion in #160

# Checklist

- [ ] I've added my name to the `AUTHORS` file, if it wasn't already present.

### CODE REVIEW

**Comment by Kshitij09** [comment]
Let me know if I should Add name in the `AUTHORS` file.

**Comment by javatarz** [comment]
You should add your name @Kshitij09. Also, can you resolve conflicts please?

**Comment by javatarz** [comment]
Also, there aren't any tests for the method you just added :(

**Comment by Kshitij09** [comment]
Okay. I'll do that soon

**Comment by Kshitij09** [comment]
I'm a bit confused about the caller and parameter for `shouldBeInRange` assertion.

Referring to issue #119  from the parent repo, it's mentioned as:

```
val r1 = 1..9
val r2 = 4..5

r1 shouldBeInRange r2 //fail
r2 shouldBeInRange r1 //pass
```

whereas, in your part of implementation, it works for exact opposite

```
val targetRange: IntRange = 1..9
val inputRange = 4..5

targetRange.shouldBeInRange(inputRange)  // pass
inputRange.shouldBeInRange(targetRange) // fail
```

So I'm not sure what should be in whose range? please clarify the same.

**Comment by Kshitij09** [comment]
# Checklist

- [x] I've added my name to the AUTHORS file, if it wasn't already present.

File: AUTHORS.md [hunk 1]
```diff
@@ -35,3 +35,4 @@
 1. Jógvan Olsen - [@jeggy](https://github.com/jeggy) ([Contributions](https://github.com/MarkusAmshove/Kluent/commits?author=jeggy))
 1. Yang C - [@ychescale9](https://github.com/ychescale9) ([Contributions](https://github.com/MarkusAmshove/Kluent/commits?author=ychescale9))
 1. Christian Ivicevic - [@ChristianIvicevic](https://github.com/ChristianIvicevic) ([Contributions](https://github.com/MarkusAmshove/Kluent/commits?author=ChristianIvicevic))
+1. Kshitij Patil [@Kshitij09](https://github.com/Kshitij09) ([Contributions](https://github.com/MarkusAmshove/Kluent/commits?author=ChristianIvicevic))
```
Comment (javatarz) [inline AUTHORS.md]: You probably want to link to https://github.com/MarkusAmshove/Kluent/commits?author=Kshitij09 :)

**Review by javatarz — CHANGES_REQUESTED** [review]

**Review by javatarz — APPROVED** [review]

**Pull request merged**